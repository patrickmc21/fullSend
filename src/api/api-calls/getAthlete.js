export const redirectLogin = () => {
  window.location = 'https://www.strava.com/oauth/authorize?client_id=24677&response_type=code&redirect_uri=http://localhost:3000&approval_prompt=force';
}