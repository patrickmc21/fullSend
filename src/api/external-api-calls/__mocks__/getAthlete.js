export const redirectLogin = jest.fn();