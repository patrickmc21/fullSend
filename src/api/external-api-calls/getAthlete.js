export const redirectLogin = () => {
  const url = 'https://www.strava.com/oauth/authorize';
  const urlOptions = '?client_id=24677&response_type=code&';
  const redirect = 'redirect_uri=http://http://fullsend.surge.sh/&approval_prompt=force';
  window.location = `${url}${urlOptions}${redirect}`;
};