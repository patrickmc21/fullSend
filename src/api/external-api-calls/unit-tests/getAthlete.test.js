import { redirectLogin } from '../getAthlete';

describe('redirectLogin', () => {
  it('should be defined', () => {
    expect(redirectLogin).toBeDefined();
  });
});