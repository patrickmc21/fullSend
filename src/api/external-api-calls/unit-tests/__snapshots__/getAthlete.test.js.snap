// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redirectLogin should match the snapshot 1`] = `[Function]`;
